packages/admin/resources/views/components/stats/card.blade.php

```diff
@@ -57,56 +57,73 @@
     </div>
 
     @if ($chart)
-        <div class="absolute bottom-0 inset-x-0 rounded-b-2xl overflow-hidden">
-            <canvas
-                x-data="{
-                    chart: null,
+        <div
+            x-title="card-chart"
```

Comment:
What is this `x-title` doing?

Reply:
If you use someting like [Alpine.js devtools](https://chrome.google.com/webstore/detail/alpinejs-devtools/fopaemeedckajflibkpifppcankfmbhk) it gives you clear name of component for better indication.

In this example we got "card-chart" instead of just "canvas":

![screen](https://user-images.githubusercontent.com/3818874/173249117-b0c177e1-0270-4da3-a330-2e0d256f1862.jpg)